tasks/utilities/submission/task_submission.wdl

```diff
@@ -60,62 +60,63 @@ task prune_table {
-      # add a column for biosample package -- required for XML submission
-      table["attribute_package"] = "SARS-CoV-2.wwsurv.1.0"
-
-      # qc checks:
-
-   
-    elif ("~{biosample_type}".lower() == "pathogen") or ("pathogen.cl" in "~{biosample_type}".lower()):
-      required_metadata = ["submission_id", "organism", "collected_by", "collection_date", "geo_loc_name", "host", "host_disease", "isolation_source", "lat_lon"]
-      optional_metadata = ["sample_title", "isolation_type", "bioproject_accession", "attribute_package", "strain", "isolate", "culture_collection", "genotype", "host_age", "host_description", "host_disease_outcome", "host_disease_stage", "host_health_state", "host_sex", "host_subject_id", "host_tissue_sampled", "passage_history", "pathotype", "serotype", "serovar", "specimen_voucher", "subgroup", "subtype", "description"] 
-      # add a column for biosample package -- required for XML submission
-      table["attribute_package"] = "Pathogen.cl"
-      # future qc checks:
-      #   gc after trimming 42-47.5%
-      #   average phred after trimming >= 28
-     
-      #   coverage after trimming >= 20X
-      #if "mean_coverage_depth" in table.columns:
-      #  table = table[(table.mean_coverage_depth > 20)]
-    elif ("pathogen.env" in "~{biosample_type}".lower()):
-      required_metadata = ["submission_id", "organism", "collected_by", "collection_date", "geo_loc_name", "isolation_source", "lat_lon"]
-      optional_metadata = ["host", "host_disease", "isolation_type", "sample_title", "bioproject_accession", "attribute_package", "strain", "isolate", "culture_collection", "genotype", "host_age", "host_description", "host_disease_outcome", "host_disease_stage", "host_health_state", "host_sex", "host_subject_id", "host_tissue_sampled", "passage_history", "pathotype", "serotype", "serovar", "specimen_voucher", "subgroup", "subtype", "description"] 
-      # add a column for biosample package -- required for XML submission
-      table["attribute_package"] = "Pathogen.env.1.0"
-      # future qc checks:
-      #   gc after trimming 42-47.5%
-      #   average phred after trimming >= 28
-     
-      #   coverage after trimming >= 20X
-      #if "mean_coverage_depth" in table.columns:
-      #  table = table[(table.mean_coverage_depth > 20)]
-
-    elif ("~{biosample_type}".lower() == "virus"):
-      required_metadata = ["submission_id", "organism", "isolate", "collection_date", "geo_loc_name", "isolation_source"]
-      optional_metadata = ["sample_title", "bioprojection_accession","attribute_package", "host", "lab_host", "altitude", "biomaterial_provider", "collected_by", "culture_collection", "depth", "disease", "env_broad_scale", "genotype", "host_tissue_sampled", "identified_by", "lat_lon", "passage_history", "samp_size", "serotype", "specimen_voucher", "strain", "temp", "description"]
-
-      table["attribute_package"] = "Virus.1.0"
+    if (os.environ["skip_bio"] == "false"):
```

Comment:
Seems to be an upstream issue with the `skip_bio` flag: in line 33 above, looks as if any value, even `"false"` would be converted to a `true` boolean value. @sage-wright am I reading this correctly?

Reply:
No, the `skip_bio` flag would not be converted to a true Boolean even if it was false